### PR TITLE
fix: copy in mprocs

### LINKED ISSUE #2300: Newlines added when copying multi-line text in mprocs
First, run mprocs with `just fed-shell`. Then, generate a long string that spans multiple lines, select it, and then press "c" to copy it:

image

When I paste this string, I get the following text with linebreaks:

```
AAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABCEugU0FOLagwr2NaNai2VNvDxe0vfNXss7bfggLQVK2wa1W5
PmtX7Dm72XyraGBSljmIadzJagAwofdsWXYDdPi0ErRGfwOs/Qp9D9RZYLGetUCm46xl3Z6dR0f55+RGSLQEJ9atIW9
JJmJOPYc/lg==
```

So I can't just paste the output back into the `fedimint-cli reissue` command without manually editing the linebreaks.

[Asked this question upstream in mprocs repo, too](https://github.com/pvolok/mprocs/issues/79).

### PR DESCRIPTION
**Pull request by Maan2003**
fixes #2300

### CODE REVIEW

**Comment by justinmoon** [comment]
OMG IT WORKS ON MAC!

**Comment by justinmoon** [comment]
I tested on mac, manmeet tested on wayland and x11. Ship it.

**Comment by Maan2003** [comment]
likely doesn't work on replit or WSL.

**Review by justinmoon — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -328,7 +328,17 @@
...
+                    xclip
+                    wl-clipboard
+                    (mprocs.overrideAttrs (final: prev: {
+                      patches = prev.patches ++ [
+                        (fetchurl {
+                          url = "https://github.com/pvolok/mprocs/pull/88.patch";
+                          name = "clipboard-fix.patch";
+                          sha256 = "sha256-9dx1vaEQ6kD66M+vsJLIq1FK+nEObuXSi3cmpSZuQWk=";
+                        })
+                      ];
```
Comment (elsirion) [inline flake.nix:340]: The power of Nix :exploding_head:

**Pull request merged**